src/main/java/bio/terra/cli/serialization/userfacing/resource/UFBqDataset.java

```diff
@@ -40,12 +47,14 @@ public void print(String prefix) {
     PrintStream OUT = UserIO.getOut();
     OUT.println(prefix + "GCP project id: " + projectId);
     OUT.println(prefix + "BigQuery dataset id: " + datasetId);
+    OUT.println(prefix + "Location: " + location);
```

Comment:
nit: maybe conditionally print this on a non-null `location`, or provide a non-null default? This should always be populated, but this might look strange if it isn't. Same for buckets

Reply:
Good idea. Added an `(undefined)` null display value for buckets, datasets, and notebooks.